CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- ShopgateConfigInterface::buildConfigFilePath()
+- ShopgateConfig::buildConfigFilePath()
+
+### Changed
```

Comment:
doesn't depcrecated have it's own topic?

Reply:
That's right, thanks.